apis/v1alpha2/nginxproxy_types.go

```diff
@@ -608,3 +613,16 @@ type NodePort struct {
 	// kubebuilder:validation:Maximum=65535
 	ListenerPort int32 `json:"listenerPort"`
 }
+
+// HostPort exposes an nginx container port on the host.
+type HostPort struct {
+	// Port to expose on the host.
+	// kubebuilder:validation:Minimum=1
+	// kubebuilder:validation:Maximum=65535
```

Comment:
This actually needs to have a `+` before `kubebuilder` in order to render properly. It looks like our NodePort struct has this mistake as well. Do you mind updating all of these?

`// +kubebuilder:validation:Minimum=1` for example.